packages/cli/src/cmds/preview/preview.ts

```diff
@@ -430,10 +442,14 @@ export default class Preview extends YargsCommand {
       backendExtensionsInstallTask = new Task(
         constants.backendExtensionsInstallTitle,
         false,
-        constants.backendExtensionsInstallCommand,
+        // env checker: use dotnet execPath
+        constants.backendExtensionsInstallCommand.replace(
+          "@execPath",
+          await dotnetChecker.getDotnetExecPath()
+        ),
```

Comment:
Do not forget to handle whitespace in dotnet path.

Reply:
Error due to whitespace in dotnet path.
![image](https://user-images.githubusercontent.com/37978464/125883023-a567993a-ec05-44ff-b3b0-8b03a039e4c5.png)

Reply:
Good catch, fix it later.

Reply:
Will fix in later PR?